stablehlo/reference/Ops.cpp

```diff
@@ -321,6 +321,14 @@ SmallVector<InterpreterValue> eval(Region &region,
   scope.add(block.getArguments(), args);
 
   for (Operation &operation : block) {
+    if (!llvm::all_of(operation.getResults(), [](OpResult r) {
```

Comment:
Nice. Should we also check for operands, or is that handled elsewhere?

Reply:
IMO checking the results covers this because the operands will have to be defined as the result of some op so they will be checked as well.